tests/serverless_functions/index.ts

```diff
@@ -10,15 +11,13 @@ function getContentType() {
 const testFunc = new aws.serverless.Function("f", {
   policies: [aws.iam.AWSLambdaFullAccess],
   includePaths: ['./Pulumi.yaml'],
-  includePackages: ['body-parser'],
 }, async (ev, ctx, cb) => {
   var aws = await import('aws-sdk');
   var express = await import('express');
   var os = require('os');
   var slack = require('@slack/client');
-  // TODO[pulumi/pulumi#463] Its reasonable to expect that `./other` would work here, but it currently does not.  For
-  // now, validate that the approach that does currently work will serialize the dependencies correctly.
-  var answer = require('./bin/other').answer;
+
+  var answer = other.answer;
```

Comment:
This no longer really tests any of the logic in this repo.  Should we just remove this part of the test case?